### PR TITLE
Fixes to flora data

### PR DESCRIPTION
**Pull request by dcol2804**
These are small edits ( ~100 taxa) for predominantly woodiness and plant growth form trait values which were incorrect due to not thoroughly searching and assigning values based on descriptions of woodiness in the text, or from small systematic errors in the way I inferred growth forms. After attempting to rectify the changes using code, I ended up losing other valuable trait information and so the identification of the fixes are a mixture of comparing the "patch" I created with the old files, and also manual searching for the term "wood*" in the text.

### CODE REVIEW

File: data/NHNSW_2022/data.csv [hunk 1]
```diff
@@ -4671,8 +4671,8 @@
 "Gonocarpus","Gonocarpus tetragynus"," erect or ascending perennial herb 15-30 cm high; ","perennial","perennial","","","herb","herb","","","","","erect decumbent","erect ascending","herbaceous",""
 "Gonocarpus","Gonocarpus teucrioides"," erect, multistemmed herb or subshrub, usually 20-40 cm high; ","","","","","herb subshrub","herb subshrub","","","","","erect","erect","herbaceous",""
 "Gonocarpus","Gonocarpus micranthus subsp. ramosissimus","  ","","","","","herb","inferred_from_species","","","","","","","herbaceous",""
-"Goodenia","Goodenia heterophylla subsp. heterophylla"," erect to ascending herb with simple, ","","","","","herb","herb","","","","","erect decumbent","erect ascending","herbaceous",""
-"Goodenia","Goodenia heterophylla subsp. montana"," ascending to erect herbs, usually hairy but  hairs, ","","","","","herb","herbs","","","","","erect decumbent","erect ascending","herbaceous",""
+"Goodenia","Goodenia heterophylla subsp. heterophylla"," erect to ascending herb with simple, ","","","","","herb","herb","","","","","erect decumbent","erect ascending","woody",""
```
Comment (ehwenk) [inline data/NHNSW_2022/data.csv:4674]: Where is this `woody` inferred from?

File: data/RBGV_2022/data.csv [hunk 2]
```diff
@@ -5942,8 +5942,8 @@ Securigera,Securigera varia, erect or ascending perennial herb to circa 80 cm ta
 Securigera,Securigera," annual or perennial herbs, sometimes woodybase; ",annual perennial,annual perennial,,,herb,herbs,,,,,,,woody_base herbaceous,woodybase
 Sedum,Sedum caespitosum, erect annual herb; ,annual,annual,,,herb,herb,,,,,erect,erect,herbaceous,
 Sedum,Sedum," annual or perennial soft-wooded herbs or shrubs, more or less succulent, glabrous. ",annual perennial,annual perennial,,,herb shrub,herbs shrubs,,,,,,,woody herbaceous,
-Sedum,Sedum  xrubrotinctum, tufted perennial subshrub; ,perennial,perennial,,,subshrub,subshrub,,,,,,,,
-Sedum,Sedum acre," perennial procumbent or ascending, mat-forming herb; ",perennial,perennial,,,herb,herb,,,,,mat-forming decumbent prostrate,matforming ascending procumbent,herbaceous,
+Sedum,Sedum  xrubrotinctum, tufted perennial subshrub; ,perennial,perennial,,,subshrub,subshrub,,,,,,,soft_wood,
+Sedum,Sedum acre," perennial procumbent or ascending, mat-forming herb; ",perennial,perennial,,,herb,herb,,,,,mat-forming decumbent prostrate,matforming ascending procumbent,soft_wood,
```
Comment (ehwenk) [inline data/RBGV_2022/data.csv:5946]: Can you double check this one?

**Review by ehwenk — COMMENTED** [review]
I've checked well over 50% of the line changes and only found 1 that I was unsure about, so this looks excellent! Only thing - can you remove the 4 files ".va" (read lock) files from the GitHub repository.

**Comment by dfalster** [comment]
Hi @ehwenk 

Tests are now passing and unneeded files removed. I'll let you review the actual data

**Review by ehwenk — APPROVED** [review]
This looks good now. Thanks for making the changes!

**Pull request merged**